sensu-plugins-jenkins.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsJenkins::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin',        '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin',        '~> 2.5'
```

Comment:
This is a breaking change and needs to be called out in the changelog.

Reply:
Updated the changelog accordingly.